generate-repo-cache.nu

```diff
@@ -12,8 +12,43 @@ def get-metadata [
     |from yaml
 }
 
+def check-required-attributes [
+    metadata
+] {
+    $metadata
```

Comment:
you could pipe metadata into `columns` to retrieve columns list and then do list comparison instead of comparing each entry separately

Reply:
I've changed it a bit using columns as you've said but we still have to iterate over each element, because the elements mustn't be empty